landingzones/tests/test_views.py

```diff
@@ -267,7 +267,7 @@ def test_render(self):
                     kwargs={'landingzone': self.landing_zone.sodar_uuid},
                 )
             )
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, 302)
```

Comment:
You can remove this test from `test_views` and instead add an extra test in `test_views_taskflow` to ensure the behaviour with a data object in the landing zone.